docs/01-intro-and-building-llvm.mkd

```diff
@@ -74,7 +73,7 @@ have better error messages.
 
 LLVM has an [extensive test
 infrastructure](http://llvm.org/docs/TestingGuide.html). Before getting stuck
-in to coding, it's worth familiarising yourself with the basics and being sure
+into coding, it's worth familiarising yourself with the basics and being sure
```

Comment:
I may be wrong, but I think "stuck in to" is acceptable because "stuck in" is a phrasal verb.